### DISCUSSION TITLE
libcef.dll文件

### DISCUSSION THREAD

**Discussion by baozhaode** [XuanchenLin/NanUI#373] [疑问解答 / Q&A] [answered]
if (string.IsNullOrEmpty(_libCefDir) || string.IsNullOrEmpty(_resourceDir))
        {
            // 无法找到libcef.dll文件路径或cef运行时文件结构不正确。
            throw new FileNotFoundException(Properties.Messages.Exception_LibCefNotFound);
        }
没找到这个dll

**Comment by XuanchenLin** [comment, marked as answer]
请仔细阅读文档，libcef.dll 需要安装 NetDimension.NanUI.Runtime 包。
